lib/emque/consuming/version.rb

```diff
@@ -1,5 +1,5 @@
 module Emque
   module Consuming
-    VERSION = "1.4.0"
+    VERSION = "1.5.0"
```

Comment:
@ryanwi given the problems with puma, should we be releasing this as a major version? I think the right answer is, “no” but thought it was worth a discussion.

Reply:
I was leaning towards the minor bump since puma is only used for the control/status.

Reply:
good call